docs/chymyst02.md

```diff
@@ -386,7 +386,7 @@ run { case result(n) + done(x) =>
     }
 ```
 
-To make the chemistry clearer, we can rewrite this as three reactions using pattern-matching:
+To make the chemistry clearer, we can rewrite this as three reactions using pattern matching:
```

Comment:
Well, at least this is not German, where you would just say things like "the Reactionpatternmatchingexpression for Blockingmoleculesyntax is shown in Followingtext" ;)

I am trying to see whether the reader could misunderstand the text. "Runtime" and "run time" are two different things to me: "runtime" is a library that is loaded at "run time".